aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1373,7 +1372,6 @@ impl AptosVM {
                     &simulation_vm.0.as_move_resolver(state_view),
                     txn,
                     &log_context,
-                    false,
```

Comment:
How does the simulation work now?  Let's fix this because otherwise we have some stale code that talks about "aggregator enabled" which is a non-existent thing.

Reply:
I think we panicked in case of failures there😄 the problem right now is that we re-execute but because there is no flag we will keep failing again, and again, etc.

Reply:
So, what's the solution now?